addons/l10n_pe/models/account_tax.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.tax'
+
+    l10n_pe_code = fields.Selection(
+        [('1000', 'IGV'), ('2000', 'ISC'), ('9995', 'EXP'),
+         ('9996', 'GRA'), ('9997', 'EXO'), ('9998', 'INA'),
+         ('9999', 'OTROS')], help="This codes will help to"
+        "separate the different type of taxes in Peru",
+        string='Tax code identifier, refers to Catalog N°5')
+    l10n_pe_fee_name = fields.Selection(
+        [('VAT', 'VAT'), ('EXC', 'EXC'), ('FRE', 'FRE'), ('OTH', 'OTH')],
+        help="Code to identify the type of fee group based on UN/ECE 5153",
+        string='Fee tax name code',
+        compute='_compute_tax_groups')
+    l10n_pe_category_code = fields.Selection(
```

Comment:
We might want to remove this fields and add a method that retrieve this information. There is no other use than to identify the tax, and **l10n_pe_code** is just that. The rest of the properties can be returned with a dictionary.

Reply:
Checking https://github.com/odoo/odoo/tree/12.0/addons/l10n_mx I don't thing we should add this here, this data should be in the localization like  https://github.com/odoo/enterprise/blob/12.0/l10n_mx_edi/models/account.py#L22 has done, we must only add the taxes here

Reply:
For the moment we are going to add just data to this module, not models or fields